modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,63 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
```

Comment:
Should this stay as "Uninstalling metering from OCP" since you are removing the Metering Operator _and_ all metering components? So it's more than only uninstalling the Operator?

Reply:
@lbarbeevargas @timflannagan1 actually the metering operator is isolated by namespace...so this should read `Uninstalling metering from your namespace`